schedulearn-frontend/src/components/WorkerList.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './WorkerList.css';
+
+export interface WorkerListItem {
+  userId: number;
+  fullName: string;
+  jobTitle: string;
+  dateFrom: string;
```

Comment:
Because this data comes from Dotnetify (c#) these fields must be CamelCase to be mapped correctly.